_posts/developers/apps/vocabularies/2019-01-01-04_quickstart-artifacts.md

```diff
@@ -22,7 +22,7 @@ const iri = rdf.type;
 ```
 However, an issue with these libraries is that, by their nature, they are limited to just the common, well-known vocabularies. What about your specific vocabulary, designed for your app? How can you make the terms described in that vocabulary easily reusable in your code (and easily reusable in the code of others who may wish to reuse the terms you've defined in your cool vocabulary)?
 
-The [LIT artifact generator](https://www.npmjs.com/package/@inrupt/artifact-generator) is a tool that takes any RDF vocabulary as input and automatically generates a nice source-code bundle (i.e. an artifact such as a Java JAR, or a JavaScript NPM module) that you can then easily use in your application. Let's see how it works.
+The [artifact generator](https://www.npmjs.com/package/@inrupt/artifact-generator) is a tool (*currently available as Alpha*) that takes any RDF vocabulary as input and automatically generates a source-code bundle (i.e. an artifact such as a Java JAR, or a JavaScript NPM module) that you can then use in your application. Let's see how it works.
```

Comment:
I've been capitalizing this as 'Artifact Generator' in all the README files, as the AG is a 'thing' - which do you think is better?

Reply:
By the way, this very long single line is an example of why I always try and limit the line-lengths of Markdown files to 80 characters - i.e., I have another comment on 'the next line' of text, but I have add my comment to 'this line of text' because it's all one line.
Anyway, my 2nd comment is on "and automatically generates a source-code bundle". The Artifact Generator is intended to very often ('mostly' probably in Enterprise settings) be producing multiple artifacts (i.e., for multiple different programming languages and/or RDF libraries), so I'd suggest pluralizing this, or using "and automatically generates potentially multiple source-code bundles".

Reply:
Yup. Can capitalize and pluralize in the next patch.

I hear you about line-lengths -- for restructured text, I think I limit lines to 72.  Heh, but the line length is the  least of the page's problem 😃  Eh ... some day, when other deliverables are off the plate, these pages can undergo a real re-write ... unfortunately, as the kids used to say, today is not the day 😄

Reply:
Updated per feedback!